versions-prod.cfg

```diff
@@ -8,7 +8,7 @@ AccessControl = 5.3.1
 Acquisition = 4.10
 AuthEncoding = 4.3
 BTrees = 4.10.0
-Chameleon = 3.9.1
+Chameleon = 3.10.1
```

Comment:
Thanks for fixing that, I saw the errors during testing and didn't have the time to deal with it.